packages/ui-kit-components/src/components/Button/button.test.tsx

```diff
@@ -43,20 +43,24 @@ describe('Button component tests', () => {
   });
 
   it('test focus the button', () => {
-    render(<Button>Focus Test</Button>)
+    render(<Button>Focus Test</Button>);
 
     const element = screen.getByTestId('normal');
-    
+
     const style = window.getComputedStyle(element);
```

Comment:
I think it might be good to rename the style variables.  Something like `defaultStyle` and `focusStyle` so that it's easier to read and keep track of what's what.

Reply:
Yes, I've changed them :)